### PR TITLE
Player UI, setting UI, player class

### PR DESCRIPTION
**Pull request by minhalo113**

### CODE REVIEW

**Review by ncusimano — APPROVED** [review]
Thanks!

**Review by JFong5 — APPROVED** [review]

**Review by fuzzydomo — APPROVED** [review]
yep looks great!

**Review by LostLittleKiddo — APPROVED** [review]

File: app/src/main/java/com/example/qrranger/MainActivity.java [hunk 1]
```diff
@@ -1,14 +0,0 @@
-package com.example.qrranger;
```
Comment (ajboman) [inline app/src/main/java/com/example/qrranger/MainActivity.java:1]: This is not up to date with the `main` branch. The `MainActivity.java` file in main is populated. See this: 

https://github.com/CMPUT301W23T48/QR-Ranger/blob/main/app/src/main/java/com/example/qrranger/MainActivity.java#L1

I believe you should merge main into this branch first and then we can merge this PR.

Same thing for `activity_main.xml`

**Review by ajboman — CHANGES_REQUESTED** [review]

**Review by JFong5 — APPROVED** [review]

**Review by LostLittleKiddo — CHANGES_REQUESTED** [review]
Dont delete java doc

**Review by ncusimano — CHANGES_REQUESTED** [review]
The MainActivity is deleted.

**Pull request closed without merging**